waspc/src/Wasp/Generator/SdkGenerator.hs

```diff
@@ -37,7 +37,6 @@ import qualified Wasp.Generator.DbGenerator.Auth as DbAuth
 import Wasp.Generator.DepVersions (prismaVersion, superjsonVersion)
 import Wasp.Generator.FileDraft (FileDraft)
 import qualified Wasp.Generator.FileDraft as FD
-import qualified Wasp.Generator.JsImport as GJI
```

Comment:
This is unrelated but someone left an unused import here and the compiler was complaining.